src/content/1.7/development/coding-standards/_index.md

```diff
@@ -70,6 +70,22 @@ Javascript files MUST follow the [Airbnb Javascript style guide](https://github.
 HTML, CSS (Sass), Twig and Smarty files MUST follow the [Mark Otto's coding standards](http://codeguide.co/).
 Mark is the creator of the [Bootstrap framework](https://getbootstrap.com/).
 
+To help developers to comply with these conventions, [Stylelint](https://stylelint.io/) has been configured in the PrestaShop project. You can find the configuration file [here](https://github.com/PrestaShop/stylelint-config).
+
+Same as if you want to [compile assets]({{< ref "/1.7/development/compile-assets.md" >}}), you need NodeJS and NPM to run Stylelint.
+
+Starting on {{< minver v="1.7.8" >}}, you can run the linter like this:
+
+```bash
+npm run scss-lint
+```
+
+You can fix errors using this command:
```

Comment:
That fix all errors or just these which are fixable ?

Reply:
```suggestion
You can fix auto-fixable errors using this command:
```

What do you think?

Reply:
:ok_hand: